server/controllers/linktaFlowController.ts

```diff
@@ -29,6 +33,10 @@ const createLinktaFlowController = (
         'Fetching LinktaFlow for userInputId:',
         req.params.userInputId,
       );
+      // Check for authorization header. If not provided, throw 401 error
```

Comment:
The check for auth header in this controller is redundant here as it is already checked in the auth middleware.

Reply:
Sorry just read your test instructions and realized you might have placed the check here for testing purposes.

Reply:
thanks, removed redundant check in all files